jaeger/charts/linkerd-jaeger/templates/psp.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: psp
+  namespace: {{.Values.namespace}}
+  labels:
+    linkerd.io/extension: jaeger
+rules:
+- apiGroups: ['policy', 'extensions']
+  resources: ['podsecuritypolicies']
+  verbs: ['use']
+  resourceNames:
+  - linkerd-linkerd-control-plane
```

Comment:
This assumes the control plane's namespace is the default `linkerd`. So to make this more general you'll have to introduce the `linkerdNamespace` entry in the extension's `values.yaml`, just like we have in viz.